stories/core/Buttons/ButtonGroup.stories.tsx

```diff
@@ -77,6 +77,7 @@ export const Overflow: Story<ButtonGroupProps> = (args) => {
           maxWidth: 'clamp(300px, 50%, 100%)',
           border: '1px solid hotpink',
           padding: 8,
+          height: 'max(200px, 30vh)',
```

Comment:
I think there's some problems with this story:

1. I can see why you wouldn't want to add another story for just vertical overflow, but it's not clear that the extra height is meant to be used to look at the overflow of a vertical button bar. I would add a note like the one already there ("Resize the viewport to see overflow behavior.") to say this explicitly.
2. Because of the note, I thought I would be able to resize the vertical box as well. I think this should be added in, or the vertical overflow should have its own story, perhaps included as part of the "Vertical" story.
3. The horizontal story resizes when the box becomes smaller, but does not resize when the box becomes bigger.

https://user-images.githubusercontent.com/37936169/158854082-8fb4dc14-24e4-4c08-9561-b80234a80769.mp4